src/App.jsx

```diff
@@ -41,9 +124,37 @@ class App extends Component {
     firebaseWrapper.logout();
   };
 
-  render() {
-    const { userId } = this.state;
+  makeNotification = (type, input) => {
+    const time = `At ${new Date().toLocaleString("en-GB")},`;
+    let image = PlaceholderImage;
+    let text = input;
+    let color = "orange";
+    if (type === "makePost") {
+      text = `${text} made a new post!`;
+      color = "blue";
+      image = PostImage;
+    }
+    /* this.setState(prevState => {
```

Comment:
sent you a message about this, but just choose a way to do it that you prefer and go with it, then remove commented code